libr/anal/p/anal_x86_cs.c

```diff
@@ -1185,6 +1185,26 @@ static void anop_esil (RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len
 		}
 		break;
 	case X86_INS_MUL:
+		{
+			char *src = getarg (&gop, 0, 0, NULL);
+			char *dst;
+			switch (src[0]) {
```

Comment:
I wondered whether to handle that or not. Eventually, since I based the code on the implementation of IMUL (as well as the old implementation of MUL, as it appears in the rest of the different MUL types right below my code), I decided to follow the same tests as those two (which is, incidentally, none).